src/SwedbankPay.Sdk/PaymentOrders/PaymentOrder.cs

```diff
@@ -21,19 +22,19 @@ public class PaymentOrder
                 switch (httpOperation.Rel.Value)
                 {
                     case PaymentOrderResourceOperations.CreatePaymentOrderCapture:
-                        operations.Capture = async payload => await client.SendHttpRequestAndProcessHttpResponse<CaptureResponse>(httpOperation.Request.AttachPayload(payload));
+                        operations.Capture = async payload => await client.PostAsJsonAsync<CaptureResponse>(httpOperation.Href, payload);
                         break;
                     case PaymentOrderResourceOperations.CreatePaymentOrderCancel:
-                        operations.Cancel = async payload => await client.SendHttpRequestAndProcessHttpResponse<CancellationResponse>(httpOperation.Request.AttachPayload(payload));
+                        operations.Cancel = async payload => await client.PostAsJsonAsync<CancellationResponse>(httpOperation.Href, payload);
                         break;
                     case PaymentOrderResourceOperations.CreatePaymentOrderReversal:
-                        operations.Reversal = async payload => await client.SendHttpRequestAndProcessHttpResponse<ReversalResponse>(httpOperation.Request.AttachPayload(payload));
+                        operations.Reversal = async payload => await client.PostAsJsonAsync<ReversalResponse>(httpOperation.Href, payload);
                         break;
                     case PaymentOrderResourceOperations.UpdatePaymentOrderUpdateOrder:
-                        operations.Update = async payload => await client.SendHttpRequestAndProcessHttpResponse<PaymentOrderResponse>(httpOperation.Request.AttachPayload(payload));
+                        operations.Update = async payload => await client.PatchAsJsonAsync<PaymentOrderResponse>(httpOperation.Href, payload);
```

Comment:
This does not follow the operations design. Here we assume it's a Patch instead of using the Method from the Operations response.